Add more docs to our blog section (#942)
2i2c-org/team-compass: communication/blog.md

- If your post requires images, put them in a sub-folder within the post's folder called `images/` and refer to them with relative links from the post.

### How to add a featured image

You can optionally add a featured image, but don't worry about this unless you really want to. T do so, put an image in the same folder as the blog post that starts with `featured`.

For example, if your blog post lives at `content/blog/2024/mypost/index.md`, then you could put the featured image at `content/blog/2024/mypost/featured.png`.

### How to preview your blog post

_Note: you don't actually need to preview the post if it's too much hassle. Just make a PR and the GitHub PR preview can be used to see what it looks like._

To preview your post locally:

- Make sure you've saved the post to disk.
- Ensure that the **date** of the post is before today's date, otherwise the post won't show up (if the post will go up in the future, just temporarily make the date earlier to ensure it shows up)
- [Install Hugo](https://gohugo.io/installation/)
- Run `hugo serve -D` from the repository root

This will run a server that can be used to inspect your post.

> ### How to do all of this via GitHub in the browser without doing anything locally

Author: ```suggestion
### How to edit the blog in the browser without doing anything locally
```
